### PR TITLE
Update all our activated types to take ILogger<T> in Templates - 10332

### PR DESCRIPTION
**Pull request by isaacrlevin**
Summary of the changes (Less than 80 chars)
 - Updated all Types to have ILogger<T> wired up

Addresses #10332

### CODE REVIEW

**Comment by isaac2004** [comment]
@ryanbrandenburg Any idea what this conflict is referring to? I have tried to resolve and it isn't taking

File: src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -34,9 +34,12 @@
             </div>
         </nav>
     </header>
-    <main role="main" class="pb-3">
+    <div class="container">
```
Comment (ryanbrandenburg) [inline src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Shared/_Layout.cshtml]: How does this change relate and what is it trying to accomplish?

**Review by ryanbrandenburg — APPROVED** [review]
Over-all looks good. Once I know why the changes were made to the cshtml files this will be good to merge.

**Comment by isaac2004** [comment]
It was a regression from another one of my PR's, I noticed that the UI did weird things without those divs

**Comment by ryanbrandenburg** [comment]
Ah, makes sense.

**Pull request merged**